src/test/java/org/assertj/core/architecture/ArchitectureTest.java

```diff
@@ -0,0 +1,109 @@
+package org.assertj.core.architecture;
+
+import com.tngtech.archunit.base.DescribedPredicate;
+import com.tngtech.archunit.core.domain.AccessTarget;
+import com.tngtech.archunit.core.domain.JavaMember;
+import com.tngtech.archunit.core.domain.properties.HasName.AndFullName;
+import com.tngtech.archunit.junit.AnalyzeClasses;
+import com.tngtech.archunit.junit.ArchTest;
+import com.tngtech.archunit.lang.ArchRule;
+
+import java.net.URL;
+import java.util.Optional;
+import java.util.function.Predicate;
+
+import static com.tngtech.archunit.base.DescribedPredicate.alwaysTrue;
+import static com.tngtech.archunit.library.Architectures.layeredArchitecture;
+import static java.util.Optional.empty;
+import static java.util.Optional.of;
+import static org.assertj.core.architecture.ArchitectureTest.ROOT_PACKAGE;
+
+@AnalyzeClasses(packages = ROOT_PACKAGE)
```

Comment:
it looks it is possible to ignore test classes with ImportOption:
```
@AnalyzeClasses(packages = ROOT_PACKAGE, importOptions = ImportOption.DoNotIncludeTests.class)
```
if that works, it would simplify this test quite a bit, let's try!

Reply:
It works